src/Native/Console.js

```diff
@@ -10,10 +10,22 @@ Elm.Native.Console.make = function(localRuntime) {
 		return localRuntime.Native.Console.values;
 	}
 
+	var NS = Elm.Native.Signal.make(localRuntime);
 	var Show = Elm.Native.Show.make(localRuntime);
 	var Task = Elm.Native.Task.make(localRuntime);
 	var Utils = Elm.Native.Utils.make(localRuntime);
 
+	var readline = require('readline');
+	var rl = readline.createInterface({
+		input: process.stdin,
+	});
+	
+	var stdin = NS.input("stdin", "");
+	
+	rl.on('line', function(line) {
+		localRuntime.notify(std.id, line);
```

Comment:
Oops, that's a typo that I somehow fixed locally but forgot to commit. It's supposed to say `stdin.id` and it's a property of the `stdin` signal as creating by `Elm.Native.Signal.input`